content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx

```diff
@@ -339,6 +340,57 @@ widget.putMessage(richMessage);
   Read more about Rich Messages
 </SectionLink>
 
+## Fullscreen widgets
```

Comment:
We call them Fullscreen apps throughout the documentation so maybe we should stick to this name.

Reply:
I second this! 🙌